artiq/coredevice/comm_moninj.py

```diff
@@ -51,16 +51,20 @@ def __init__(self, monitor_cb, injection_status_cb, disconnect_cb=None):
             del self._reader
             del self._writer
 
-    def monitor(self, enable, channel, probe):
+    def monitor_probe(self, enable, channel, probe):
         packet = struct.pack(">bblb", 0, enable, channel, probe)
         self._writer.write(packet)
 
+    def monitor_injection(self, enable, channel, overrd):
+        packet = struct.pack(">bblb", 1, enable, channel, overrd)
+        self._writer.write(packet)
+
     def inject(self, channel, override, value):
-        packet = struct.pack(">blbb", 1, channel, override, value)
+        packet = struct.pack(">blbb", 2, channel, override, value)
```

Comment:
Can we avoid breaking compatibility with the current protocol?

Reply:
(to handle a dashboard/firmware version mismatch gracefully)

Reply:
OK - permuting the packet numbering would allow new firmware with the old dashboard. New dashboard and old firmware would always break things though (firmware aborts connection on unknown packet type).

Reply:
OK, that's acceptable.

Reply:
This also prevents any unexpected injections at dashboard startup.